modules/pool/pool.gql

```diff
@@ -801,28 +791,78 @@ enum GqlPoolStakingGaugeStatus {
 input GqlPoolJoinExitFilter {
     poolIdIn: [String!]
     chainIn: [GqlChain!]
+    userAddress: String
```

Comment:
Gotta tripple check if any of these changes break the current UI. I think they dont, but need to check it!

Reply:
i was paying attention to just adding new attributes to overlap join/exits with swaps